ui/apps/platform/src/Containers/Clusters/ClustersPage.tsx

```diff
@@ -32,8 +33,8 @@ function ClustersPage(): ReactElement {
 
     // Handle changes to the currently selected deployment.
     const setSelectedClusterId = useCallback(
-        (newCluster: any) => {
-            const newClusterId = newCluster?.id || newCluster || '';
+        (newCluster: string | Cluster) => {
+            const newClusterId = typeof newCluster === 'string' ? newCluster : newCluster?.id ?? '';
```

Comment:
Good question. In best case, yes.

Because I do not understand which calls provide which type, and because my first draft caused an integration test failure, I kept optional chaining and replaced `||` with `??` operator.